include/tvm/topi/nn.h

```diff
@@ -642,6 +643,54 @@ inline tvm::te::Tensor batch_to_space_nd(const tvm::te::Tensor& data,
   out = strided_slice(out, begin_idx, end_idx, strides);
   return out;
 }
+
+/*!
+ * \brief Negative log likelihood loss.
+ *
+ * \param input The input tensor.
+ * \param target The target tensor.
+ * \param weight A manual rescaling weight given to each class.
+ * \param reduction The reduction method to apply to the output.
+ * \param ignore_index The target value to ignore.
+ * \param name The name of the operation.
+ * \param tag The tag to mark the operation.
+ *
+ * \return A Tensor whose op member is the batch_to_space_nd operation
+ */
+inline Tensor nll_loss(const Tensor& input, const Tensor& target, const Tensor& weight,
+                       std::string reduction = "mean", int ignore_index = -100,
```

Comment:
What is the significance of the -100 for `ignore_index`?

Reply:
I believe this is the default used in PyTorch, there's no inherent significance to it (although certain data loading/preprocessing scripts seem to try and match this value - e.g. I've see this in certain BERT training data loaders)

Reply:
Yes, it's the default value used in pytorch, which is mainly a random negative number in my opinion.